### ISSUE TITLE
No way to reset report for a website

### ISSUE THREAD

**Issue by ecollart2** [siteinspector/siteinspector#46]
Hello,

I started a scan for a website but forgot to remove spelling checks (site in French) so I stopped the scan and modified the configuration but:

1.  the errors from previous scan are still part of the report
2. removing and adding website doesn't reset report either.

Thanks for this very nice application !

Eric Collart

**Comment by AdamDudley** [comment]
I would second this

version v1.2.0 (Docker)

Confusing for the user, I attempted to 'reset' a report by removing it and readding it.

I now have to 're add it' every time I want to see it!

Steps
1. Create new site/url
2. start scan
3. while its still scanning, remove the site (to give an example of site size, I had about 6000 spelling errors
4. re add the site

expected:
new site, with 0 results ready to press start scan

actual: 
existing items where shown, although the number of spellings were going down (I had fixed 6 that were repeated on many pages)

Further comments

When re-visiting the UI from fresh login, the above site is not visible, you have to 're add' it each time and then it shows, with the current status etc.

**Comment by AdamDudley** [comment]
I think 'reset' should just be in the 'setting' section, and that would delete all entries.

I think to fix the bug I describe, when adding a site, always set public.websites.status text to 'enabled' regardless will solve the issues. manually changing this in the database has 'fixed' the problem of the site not showing unless I re-add it.

**Comment by israelcurtis** [comment]
Very much want this feature as well, fairly surprised that there's no way to get rid of old entires even when they've been "marked as resolved"!
we need to periodically check the same sites over and over, as their content gets updated

**Comment by davidbraw** [comment]
Yes, I've got this problem too. Tried to remove a sit and readd it as certain fixed issue didn't go away and thought there was some sort of caching happening. AdamDudley summarised it perfectly.
